### ISSUE TITLE
Parser fail when an ELSE is empty

### ISSUE THREAD

**Issue by maxime645** [TypeCobolTeam/TypeCobol#1460]
**Describe the bug**
When an IF or an ELSE is empty, the parser isn't able to parse the document correctly. The rest of the document will be recognized as child of a child of the last node in the current program.

**To Reproduce**
(Type)Cobol code that cause the bug : (if any)
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. DVZZBCO0.
       ENVIRONMENT DIVISION.
      
       CONFIGURATION SECTION.
      *SOURCE-COMPUTER.    IBM-3033 WITH DEBUGGING MODE.
       SPECIAL-NAMES.      DECIMAL-POINT IS COMMA.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       01 STR.
           10 MyBool TYPE BOOL.
           10 MyChar PIC X.

       procedure division.
           IF STR::MyChar = SPACE
                MOVE 'X' TO STR::MyChar
           ELSE
           END-IF
           GOBACK.
      
       DECLARE PROCEDURE MyProc.
       END-DECLARE.
      
       END PROGRAM DVZZBCO0.
```
You can see in the TypeCobol Outline that MyProc is child of DVZZBCO0\Procedure Division\If\Then while it should be child of DVZZBCO0\Procedure Division

**Comment by fm-117** [comment]
Duplicate with #1660 and #1676 ?

**Comment by smedilol** [comment]
It's fixed by #1676
